projects/packages/blaze/src/class-blaze.php

```diff
@@ -121,6 +121,11 @@ public static function should_initialize() {
 		$connection        = new Jetpack_Connection();
 		$site_id           = Jetpack_Connection::get_site_id();
 
+		// Only admins should be able to Blaze posts on a site.
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return false;
```

Comment:
In order to not override the filter: 
```suggestion
			$should_initialize = false; 
```

Reply:
That's what I had done at first, but I didn't want to run the other checks for a role where we're sure we do not want to show the UI.

Reply:
ok! I'm out of the loop, why are we so sure?

Reply:
We talked about adding the admin condition in peeHDf-zm-p2